tools/distrib/python/grpcio_tools/setup.py

```diff
@@ -95,7 +95,10 @@ def protoc_ext_module():
           protoc_lib_deps.CC_INCLUDE,
       ],
       language='c++',
-      define_macros=[('HAVE_PTHREAD', 1)],
+      define_macros=[
+          ('HAVE_PTHREAD', 1,),
+          ('GOOGLE_PROTOBUF_NO_THREADLOCAL', 1,),
```

Comment:
We could. But this is just the code generator (performance isn't super important), and the protobuf runtime already falls back to pthread-based TLS if it doesn't have access to `__thread` anyway, so I don't see any specific harm in setting it. Am I missing anything?